components/financial_assistance/app/controllers/financial_assistance/incomes_controller.rb

```diff
@@ -6,8 +6,8 @@ class IncomesController < FinancialAssistance::ApplicationController
 
     before_action :find_application_and_applicant
     before_action :set_cache_headers, only: [:index, :other]
-    before_action :enable_bs4_layout, only: [:other] if EnrollRegistry.feature_enabled?(:bs4_consumer_flow)
-    before_action :conditionally_enable_bs4_layout, only: [:create, :update] if EnrollRegistry.feature_enabled?(:bs4_consumer_flow)
+    before_action :enable_bs4_layout, only: [:other, :index, :create] if EnrollRegistry.feature_enabled?(:bs4_consumer_flow)
+    before_action :conditionally_enable_bs4_layout, only: [:new, :create, :edit, :update] if EnrollRegistry.feature_enabled?(:bs4_consumer_flow)
```

Comment:
do `new` and `edit` need to be declared here? I don't think they're rendering any UI

Reply:
New and edit render js that switched out the form vs the show for the income elements

Reply:
@kristinmerbach on my local, it doesn't look like these endpoints are even called. I think they were switched out at some point for some in-place JS

Reply:
removed both as it was working fine on local without either